src/includes/performance/control-data-truncation/javascript.mdx

```diff
@@ -11,7 +11,7 @@ const parameters = {
 };
 
 const span = transaction.startChild({
-  op: "request",
+  op: "http",
```

Comment:
```suggestion
  op: "http.client",
```

We use `http.server` for incoming requests and `http.client` for outgoing requests.